docs/src/embeddings/default_embedding_functions.md

```diff
@@ -289,4 +289,65 @@ print(actual.label)
 
 ```
 
+### Imagebind embeddings
+We have support for [imagebind](https://github.com/facebookresearch/ImageBind) model embeddings. It is registered as `imagebind` and supports Audio, Video and Text modalities(extending to Thermal,Depth,IMU data):
```

Comment:
Users still need to install this package from your fork right? Can you just publish a pypi package under some name like "imagebind-packaged" etc. and add an instruction here?

Reply:
yeah will be publishing that soon

Reply:
@raghavdixit99 any updates on this?